lib/plugins/aws/package/compile/functions.js

```diff
@@ -650,38 +648,78 @@ class AwsCompileFunctions {
     cfResources[this.provider.naming.getLambdaEventConfigLogicalId(functionName)] = resource;
   }
 
+  getDesinationsArn(destinationsProperty) {
+    if (typeof destinationsProperty === 'object') {
+      if (!destinationsProperty.arn) {
```

Comment:
After thinking about it more, I think we can skip these checks and errors as it's syntax that is not recognized by our validation schema. I'll bring this up internally tomorrow and will report back

Reply:
OK. I wait for the response.

Reply:
Let's skip these checks and rely only on schema validation 👍 Thanks for patience on this one 🙇

Reply:
OK